object/service.proto

```diff
@@ -218,6 +219,9 @@ service ObjectService {
   // - **TOKEN_EXPIRED** (4097, SECTION_SESSION): \
   //   provided session token has expired.
   rpc GetRangeHash(GetRangeHashRequest) returns (GetRangeHashResponse);
+
+  // TODO: docs
+  rpc Replicate(ReplicationRequest) returns (ReplicationResponse);
```

Comment:
Will it work for objects larger than a single grpc message?

Reply:
yes i've tested this within https://github.com/nspcc-dev/neofs-node/pull/2674/commits/62d020c8e62b805071bc7f9742b28d0c4129c60a